x-pack/plugins/cases/server/services/cases/transform.ts

```diff
@@ -192,6 +192,11 @@ export function transformSavedObjectToExternalModel(
   const status =
     STATUS_ESMODEL_TO_EXTERNAL[caseSavedObject.attributes?.status] ?? CaseStatuses.open;
 
+  // @ts-ignore
+  delete caseSavedObject.attributes?.total_alerts;
+  // @ts-ignore
+  delete caseSavedObject.attributes?.total_comments;
```

Comment:
Shouldn't we do the same for the `transformUpdateResponseToExternalModel`?

Reply:
Yup, just changed it. I had added tests for `patch` but there was a bug in them. I fixed them, added one for `bulkPatch` too, and changed `transformUpdateResponseToExternalModel` 👍

Reply:
Amazing! I was wondering why the tests did not catch it 🙂.